libs/shared/ui/src/lib/components/accordion/accordion.tsx

```diff
@@ -49,14 +49,14 @@ const AccordionContent = forwardRef<ElementRef<typeof AccordionPrimitive.Content
     <AccordionPrimitive.Content
       className={twMerge(
```

Comment:
I removed the children wrapper to allow the possibility to override every class name